pkgs/development/python-modules/npyscreen/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "npyscreen";
+  version = "4.10.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vhjwn0dan3zmffvh80dxb4x67jysvvf1imp6pk4dsfslpwy0bk2";
+  };
+
+  doCheck = false;
```

Comment:
if tests aren't bundled with the pypi tarball, please grab from github https://github.com/npcole/npyscreen.

Tests are crucial to ensure that the package still works. You can disable the tests which don't work well within a sandbox